extractor.js

```diff
@@ -170,7 +173,8 @@ function parseRegionsFile(buffer) {
 			}
 		};
 
-		region.id = stream.readUInt32BE();
+		region.full_id = stream.readUInt32BE();
+		region.id = cleanRegionId(region.full_id);
```

Comment:
The function can be removed from this and just in-lined, and also simplified to just `(value >> 16) & 0xFF`. There are no regions which go past this so it should be fine